alacritty_terminal/src/grid/mod.rs

```diff
@@ -187,11 +187,6 @@ impl<T: GridCell + Default + PartialEq + Copy> Grid<T> {
         Point { line: self.lines.0 + self.display_offset - point.line.0 - 1, col: point.col }
     }
 
-    /// Return the cursor position in buffer coordinates.
-    pub fn cursor_buffer_point(&self) -> Point<usize> {
-        Point { line: self.lines.0 - self.cursor.point.line.0 - 1, col: self.cursor.point.col }
-    }
```

Comment:
I've considered what to do with this function and decided to remove it.

If we'd want to keep the function, we'd have to basically create the same function but for the saved cursor, which seems a bit annoying. Alternatively we could of course create a generic function and pass a point to it, but I feel like at that point it's easier to just compute things ourselves. Especially because **all** instances where we need this currently only require the line, not the column.

What do you think @kchibisov?

Reply:
It fine to just remove it.